apps/skde/src/styles/globals.css

```diff
@@ -19,26 +19,6 @@ a {
   box-sizing: border-box;
 }
 
-h1 {
-  font-weight: 300;
-  font-size: 48px;
-}
-
-h2 {
-  font-weight: 600;
-  font-size: 24px;
-  line-height: 24px;
-  text-transform: uppercase;
-  border-bottom: 2px solid #cddbe8;
-  padding-bottom: 15px;
-}
-
-h3 {
-  font-weight: 600;
-  font-size: 20px;
-  line-height: 24px;
-}
-
```

Comment:
Fjerning av dette føkker opp styling av helseatlas (hvis det er lov å skrive, og det tror jeg det er siden vi ikke har noen code of conduct i denne repo).

Reply:
Revertert